fe/fe-core/src/main/java/com/starrocks/connector/PartitionUtil.java

```diff
@@ -488,20 +490,41 @@ public static Map<String, Range<PartitionKey>> getRangePartitionMapOfExternalTab
 
         Map<String, Range<PartitionKey>> mvPartitionRangeMap = new LinkedHashMap<>();
         for (Map.Entry<String, PartitionKey> entry : sortedPartitionLinkMap.entrySet()) {
-            if (index == 0) {
-                lastPartitionName = entry.getKey();
+            if (entry.getValue().getKeys().get(0) instanceof StringLiteral) {
+                if (index == 0) {
+                    lastPartitionKey = entry.getValue();
+                    PartitionKey infinityPartitionKey =
+                            PartitionKey.createInfinityPartitionKey(ImmutableList.of(partitionColumn), false);
+                    mvPartitionRangeMap.put(entry.getKey(), Range.closedOpen(infinityPartitionKey, entry.getValue()));
+                    ++index;
+                    continue;
+                }
+                Preconditions.checkState(!mvPartitionRangeMap.containsKey(lastPartitionName));
+                if (entry.getValue().getKeys().get(0).getStringValue().equals("MAXVALUE")) {
```

Comment:
where does `MAXVALUE ` come from? it's better to define a `CONTANT` var for this.

Reply:
Maxvalue comes from the last partition of the MySQL partition table. To avoid errors, I added a constant.